### PR TITLE
Add Support for Flyte Scribe

### PR DESCRIPTION
**Pull request by EzzPrograms**
Hi,
I've been using PreMiD for a while and thought that support for edo/flyte's blog could be added here.

### CODE REVIEW

File: Flyte Scribe/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Ezz",
+        "id": "256009740239241216"
+    },
+    "url": "flyte.moe",
+    "description": {
+        "en": "A simple presence for edo/flyte's blog."
```
Comment (OfficialCRUGG) [inline Flyte Scribe/dist/metadata.json:8]: You should probably change the Description as it should be a Description of the Website. So e.g. "edo/flyte's blog is a Blog about [...]"

Comment (EzzPrograms) [reply Flyte Scribe/dist/metadata.json]: Added a more detailed description as requested in commit https://github.com/EzzPrograms/Presences/commit/f3f44d27829e503155bfc9c6e0b48a111d3327bd.

File: Flyte Scribe/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```
Comment (Bas950) [inline Flyte Scribe/tsconfig.json:6]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

File: Flyte Scribe/presence.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+var presence = new Presence({
+    clientId: "707632555612045413"
+  }),
+  presenceData: presenceData = {
+    largeImageKey: "logo"
+  },
+  customData = false;
+
+presence.on("UpdateData", async () => {
+  customData = false;
```
Comment (Bas950) [inline Flyte Scribe/presence.ts:10]: custromData is never set to true, so why is it needed?

File: Flyte Scribe/tsconfig.json (hunk 2 above)
Comment (OfficialCRUGG) [reply Flyte Scribe/tsconfig.json:6]: Where is the Issue?

Comment (Bas950) [reply Flyte Scribe/tsconfig.json:6]: outdir

Comment (EzzPrograms) [reply Flyte Scribe/tsconfig.json]: Fixed outDir issue in https://github.com/PreMiD/Presences/pull/1511/commits/461c43ae697bbbbe89b9dadc4990f93c039e5b50

File: Flyte Scribe/presence.ts (hunk 3 above)
Comment (EzzPrograms) [reply Flyte Scribe/presence.ts]: During development, edo/flyte had a page which had some custom data in it depending on the day.
Shortly before the pull request was made, edo had decided it was not needed there.
I somehow forgot to remove that bit of line, and fixed other lines that were dependent on this variable.
It's now fixed in https://github.com/PreMiD/Presences/pull/1511/commits/461c43ae697bbbbe89b9dadc4990f93c039e5b50.

File: Flyte Scribe/presence.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+var presence = new Presence({
+    clientId: "707632555612045413"
+  }),
+  presenceData: presenceData = {
+    largeImageKey: "logo"
+  }
```
Comment (Bas950) [inline Flyte Scribe/presence.ts:6]: you forgot a `;` at the end, it should fix your deepscan issue.

Comment (EzzPrograms) [reply Flyte Scribe/presence.ts]: Added the `;` in presence.ts and recompiled in https://github.com/PreMiD/Presences/pull/1511/commits/d0b4847cfff91e1c8b4dfb51a778f5b89868b1cd

**Comment by Bas950** [comment]
Upload proof of it working and you should be good to go :)

**Comment by EzzPrograms** [comment]
[This](https://streamable.com/qrubjk) should be the proof that it's working.

**Comment by Bas950** [comment]
> [This](https://streamable.com/qrubjk) should be the proof that it's working.

Might be nice to add a time elapsed.

**Comment by EzzPrograms** [comment]
> Set a color, and upload the proof **image** _in the PR_.

May I know exactly which color you are referring to?
This piece of code: `"color": "#FFFFFF",` is actually following the site's accent, which is white.

For the proof, here it is in picture form:
Screenshot (150)

**Comment by EzzPrograms** [comment]
> > [This](https://streamable.com/qrubjk) should be the proof that it's working.
> 
> Might be nice to add a time elapsed.

@Bas950 I've added the time elapsed as of your recommendation.

**Comment by doomlerd** [comment]
It's pretty white, maybe pick the grayish color but #FFFFFF is not really an "accent" if you think about it.

**Comment by EzzPrograms** [comment]
> It's pretty white, maybe pick the grayish color but #FFFFFF is not really an "accent" if you think about it.

Will choose a new color, maybe `#A3A3A3` would be my choice for now.

**Pull request merged**